docs/resource-sharing/os-backfill-containers.md

```diff
@@ -213,6 +213,26 @@ the `--memory` argument tells Docker to kill the container if its total memory u
     Additionally, the processes that manage jobs also use some amount of memory.
     Therefore, it is important to give the container some extra room.
 
+
+### Advanced: Advertising additional pilot attributes
+
+You can put arbitrary additional attributes in the machine ads that you advertise to the OSPool.
+These attributes will show up when you do `condor_status -l` on your machine.
+This could be useful for advertising something about the way the pilot was provisioned.
+To do this, volume-mount a file containing `key=value` pairs to `/etc/osg/extra-attributes.cfg`.
+Keys must be valid classad attribute names and values must be valid classad expressions.
+Multi-line strings are not supported.
+A line starting with `#` will be treated as a comment.
+For example:
+
+```
+# The Kubernetes namespace this pod is running under
+Kubernetes_Namespace = "path-osgdev"
```

Comment:
```suggestion
KUBERNETES_NAMESPACE = "path-osgdev"
```

Unless there's some convention for these attrs that indicate that we should use title casing